cmd/monaco/integrationtest/v2/multi_project_multi_user_test.go

```diff
@@ -0,0 +1,65 @@
+//go:build integration
+// +build integration
+
+/*
+ * @license
+ * Copyright 2023 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2
+
+import (
+	"github.com/dynatrace/dynatrace-configuration-as-code/cmd/monaco/dynatrace"
+	"github.com/dynatrace/dynatrace-configuration-as-code/cmd/monaco/integrationtest"
+	"github.com/dynatrace/dynatrace-configuration-as-code/cmd/monaco/runner"
+	"github.com/dynatrace/dynatrace-configuration-as-code/internal/idutils"
+	"github.com/dynatrace/dynatrace-configuration-as-code/pkg/client/dtclient"
+	"github.com/dynatrace/dynatrace-configuration-as-code/pkg/project/v2/topologysort"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+var multiProjectMultiUserFolder = "test-resources/integration-multi-project-multi-user/"
+var multiProjectMultiUserManifest = multiProjectMultiUserFolder + "manifest.yaml"
+
+// TestIntegrationMultiProjectMultiUser tries to upload a project that contatins two projects with
+// the exact same settings 2.0 object and verifies that deploying such a monaco configuration results in
+// two different settings objects deployed on the environment
+func TestIntegrationMultiProjectMultiUser(t *testing.T) {
```

Comment:
nit: That name seems off for what this tests (also applies to its resources folder)
IMO something like "TestSettingsInDifferentProjectsGetDifferentExternalIDs" would tell me more